File: laravel-demo/app/Http/Controllers/MiscController.php
<?php

namespace App\Http\Controllers;

use App\User;
use App\Http\Controllers\Controller;

class MiscController extends Controller
{
    public function blockui()
    {
        return view('misc.blockui', ['title' => 'BlockUI - Miscellaneous']);
    }

    public function clipboardjs()
    {
        return view('misc.clipboardjs', ['title' => 'Clipboard.js - Miscellaneous']);
    }

    public function idleTimer()
    {
        return view('misc.idle-timer', ['title' => 'Idle Timer - Miscellaneous']);
    }

    public function ladda()
    {
        return view('misc.ladda', ['title' => 'Ladda - Miscellaneous']);
    }

    public function masonry()
    {
        return view('misc.masonry', ['title' => 'Masonry - Miscellaneous']);
    }

    public function numeraljs()
    {
        return view('misc.numeraljs', ['title' => 'Numeral.js - Miscellaneous']);
    }

    public function perfectScrollbar()
    {
        return view('misc.perfect-scrollbar', ['title' => 'Perfect Scrollbar - Miscellaneous']);
    }

    public function spinkit()
    {
        return view('misc.spinkit', ['title' => 'SpinKit - Miscellaneous']);
    }

    public function vegasjs()
    {
        return view('misc.vegasjs', ['title' => 'Vegas.js - Miscellaneous']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/MiscController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers
{
    public class MiscController : Controller
    {

         public ActionResult Blockui()
        {
            return View("~/Views/Misc/Blockui.cshtml");
        }

         public ActionResult Clipboardjs()
        {
            return View("~/Views/Misc/Clipboardjs.cshtml");
        }

         public ActionResult IdleTimer()
        {
            return View("~/Views/Misc/IdleTimer.cshtml");
        }

         public ActionResult Ladda()
        {
            return View("~/Views/Misc/Ladda.cshtml");
        }

         public ActionResult Masonry()
        {
            return View("~/Views/Misc/Masonry.cshtml");
        }

         public ActionResult Numeraljs()
        {
            return View("~/Views/Misc/Numeraljs.cshtml");
        }

         public ActionResult PerfectScrollbar()
        {
            return View("~/Views/Misc/PerfectScrollbar.cshtml");
        }

         public ActionResult Spinkit()
        {
            return View("~/Views/Misc/Spinkit.cshtml");
        }

         public ActionResult Vegasjs()
        {
            return View("~/Views/Misc/Vegasjs.cshtml");
        }

    }
}
